interactions.go

```diff
@@ -194,20 +194,9 @@ type InteractionData interface {
 
 // ApplicationCommandInteractionData contains the data of application command interaction.
 type ApplicationCommandInteractionData struct {
-	ID       string                                     `json:"id"`
-	Name     string                                     `json:"name"`
-	Resolved *ApplicationCommandInteractionDataResolved `json:"resolved"`
-	Options  []*ApplicationCommandInteractionDataOption `json:"options"`
-}
-
-// ApplicationCommandInteractionDataResolved contains resolved data for command arguments.
-// Partial Member objects are missing user, deaf and mute fields.
-// Partial Channel objects only have id, name, type and permissions fields.
-type ApplicationCommandInteractionDataResolved struct {
```

Comment:
Is there any particular reason that this PR removes this type? It still looks to be relevant from Discord's docs.

Reply:
Hol' up